src/main/java/dev/jbang/dependencies/DependencyUtil.java

```diff
@@ -68,6 +73,8 @@ public static ModularClassPath resolveDependencies(List<String> deps, List<Maven
 		}
 
 		// Turn any URL dependencies into regular GAV coordinates
+		// TODO: we should also check if the root is a URL and turn it into a GAV if
```

Comment:
I don't think there should be any URLs at this point. They should all be GAvs already.